modules/tribes/client/components/TribesList.component.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styled from 'styled-components';
+
+import Tribe from './Tribe';
+import SuggestTribe from './SuggestTribe';
+
+const List = styled.ul`
+  margin: -5px -5px 15px -5px;
+  line-height: 0;
+`;
+
+const Item = styled.li`
+  line-height: 1.42857143; // set back the line-height set by bootstrap
+  padding: 5px;
+  display: inline-block;
+  vertical-align: bottom;
+  // background-color: red;
```

Comment:
Can remove commented out style

Reply:
Fixed in 9fcdd0ab1e89d508057425b433571a1394c42565